PREREQUISITES.md

```diff
@@ -89,7 +89,7 @@ sudo apt-get install -y python3-venv
 
 Also, install the following pip3 packages:
 ```bash
-pip3 install --upgrade setuptools json-rpc py-solc web3 wheel
+pip3 install --upgrade setuptools json-rpc py-solc web3 colorlog twisted wheel
```

Comment:
colorlog, twisted and wheel packages are already mentioned as part of sudo apt-get install command. No need to install them again using pip3.

Reply:
Avalon does not work unless these packages are installed with `pip3 install`. Running `apt-get install` does not work for Avalon.

Reply:
Those are well known python components. A BDBA scan of the executables will tell you if there are any outstanding CVEs in the version being pulled in.